test/models/report_test.rb

```diff
@@ -3,7 +3,32 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @user = users(:one)
+    @report = reports(:two)
+  end
+
+  test 'valid report' do
+    report = Report.new(title: 'Report one', content: 'This is report one', user_id: @user.id)
```

Comment:
```suggestion
    report = @user.reports.new(title: 'Report one', content: 'This is report one')
```

のように関連を活用した方がシンプルに書けます。